src/main/java/com/glencoesoftware/bioformats2raw/Converter.java

```diff
@@ -220,12 +222,19 @@ private static Compression getCompressor(
     PyramidTiffReader.class, MiraxReader.class
   };
 
+  @Option(
```

Comment:
We've used `--file_type <string>` on `isyntax2raw`, might be an idea to use the same here?

Ref:

 * https://github.com/glencoesoftware/isyntax2raw/blob/master/isyntax2raw/cli/isyntax2raw.py#L34